pxtblocks/blocklyloader.ts

```diff
@@ -269,7 +269,7 @@ namespace pxt.blocks {
             toolboxStyleBuffer += `
                 .blocklyTreeIcon.${className} {
                     display: inline-block !important;
-                    background-image: url("${encodeURI(i)}")!important;
+                    background-image: url("${pxt.webConfig.commitCdnUrl + encodeURI(i)}")!important;
```

Comment:
I was just thinking they shouldn't put it. The icons have to be in the CDN anyway.